### DISCUSSION TITLE
Visualizing Backtest Results

### DISCUSSION THREAD

**Discussion by tiger2014** [DaveSkender/Stock.Indicators#809] [Ideas]
Hi DaveSkender, 

This library helps me very much. And I'm thinking about how to visualize the backtesting result (like the TradeView: show the buy/sell signal and indicator on the chart). This will help us know whether the backtesting result is consistent with what we expect or not.

An MVC web might be easier to implement for someone like me (not know any front-end framework).

Thanks

**Comment by DaveSkender** [comment]
@mihakralj and I are starting to think about how to provide some additional tools -- we definitely have this in mind.  Right now, I'm focusing on this library, which is limited to producing indicators.  It doesn't include signal generation, backtesting, or provide any [visualization options](https://github.com/DaveSkender/Stock.Indicators/discussions/430).  Here's some thoughts:

- [ASP.NET Blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor) is a good UI framework to consider if you want to primarily work with our library as a front-end component.
- Our [demo site](https://stock-charts.azurewebsites.net/) is an Angular front-end with indicators generated in a backend Web API.  That code is available too, as a starting point for ideas, but really wasn't built for production quality and reusability.

**Reply by DaveSkender** [reply]
As a side note for anyone reading this, if you're interested in actively participating in the open-source development of additional community tools in the trading / financial market analysis areas, please let me know so we can include you when we're ready to start on new projects.  Serious interests only, please!  We'll need some reliable availability.

**Reply by tiger2014** [reply]
I like Blazor, and can you recommend a **free** Blazor financial chart component? 

I'm serious interest in it, for I'm set up my auto-trade system. But I can't promise the time, for I have a full-time job and a family.

**Reply by DaveSkender** [reply]
I haven't looked at charting in Blazor specifically, so I can't make a recommendation that would be better than a quick Google search.  I'll keep an eye out though.

**Reply by tiger2014** [reply]
Thanks

**Comment by tiger2014** [comment]
I found out that plotly.net is very easy to be used for showing the result of backtesting.
